view/frontend/web/js/view/payment/adapter.js

```diff
@@ -122,6 +122,10 @@ define([
         createPaymentIntent: function () {
             var self = this,
                 dfd = $.Deferred();
+            if ($("#tnw_stripe_enable_vault").length) {
+                console.log(customerData);
```

Comment:
we should not output customerData to console

Reply:
Thanks, removed